addons/miscFixes/patchCUP/config.cpp

```diff
@@ -56,6 +56,52 @@ class CfgVehicles {
             };
         };
     };
+    // Tweaks to the GTK Boxer's handling (accel/braking) + HMG swap to M3M + countermeasures move to gunner
+    class Wheeled_APC_F: Car_F {
+        class NewTurret;
+        class Turrets {
+            class MainTurret: NewTurret {
+                class Turrets {};
+            };
+        };
+    };
+    class CUP_Boxer_Base: Wheeled_APC_F {};
```

Comment:
Not writing out the full sub classes in this class results in the classes in `CUP_Boxer_Bas_HMG` inheriting from the versions from `Wheeled_APC_F`. Luckily this time it doesn't cause issues since the `Turrets` class isn't changed in `CUP_Boxer_Base`, but it should be fixed to avoid any future issues.

 @PabstMirror lmk if I got something wrong above, but that's been my understanding

Reply:
Moving to a blocking review